browser/chrome/browser/migration/migration.dtd

```diff
@@ -43,4 +44,4 @@
 <!ENTITY done.title                     "Importowanie zakończone">
 <!ENTITY done.label                     "Poniższe elementy zostały zaimportowane:">
 
-<!ENTITY closeSourceBrowser.label "Przed przejściem dalej, oroszę upewnić się, że wybrana przeglądarka jest wyłączona.">
+<!ENTITY closeSourceBrowser.label "Przed przejściem dalej proszę upewnić się, że wybrana przeglądarka jest wyłączona.">
```

Comment:
Force-pushing and history rewriting are problematic in case of already cloned repositories.

I think it is reasonable to expect that it is not the case for most pull requests before they are integrated and don't worry, it can be amended and reordered on import also.